content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,72 @@ menu:
     parent: About the project
 ---
 
+## 1.9.6 [2021-12-xx]
+
+### Features
+
+#### Backup enhancements
+
+- **Revert damaged meta nodes to a previous state**: Add the `meta-only-overwrite-force` option to [`influxd-ctl restore`](/enterprise_influxdb/v1.9/tools/influxd-ctl/#restore) to revert damaged meta nodes in an existing cluster to a previous state when restoring an InfluxDB Enterprise database.
+
+- **Estimate the size of a backup** (full or incremental) and provide progress messages. Add `-estimate` option to [`influxd-ctl backup`](/enterprise_influxdb/v1.9/tools/influxd-ctl/#backup) to estimate the size of a backup (full or incremental) and provide progress messages. Prints the number of files to back up, the percentage of bytes transferred for each file (organized by shard), and the estimated time remaining to complete the backup.
+
+#### Logging enhancements
+
+- **Log active queries when a process is terminated**: Add the [`termination-query-log = false`](/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#termination-query-log--false) configuration option. When set to `true` all running queries are printed to the log when a data node process receives a `SIGTERM` (for example, a k8s process exceeds the container memory limit or the process is terminated).
+
+- **Log all HTTP requests to meta nodes**. View updated logs when [`cluster-tracing`](/enterprise_influxdb/v1.9/administration/configure/config-meta-nodes/#cluster-tracing--false) is enabled.
+
+### Maintenance updates
+
+#### Build maintenance
+
+- Update to [Flux v0.133](/flux/v0.x/release-notes/#v01330-2021-10-04).
```

Comment:
[change log](https://dl.influxdata.com/enterprise/nightlies/master/CHANGELOG.md) says .133, is this version accurate?

Reply:
Flux v0.140.0

<pre><font color="#C4A000">commit 0541db5b72d9b60f8c7a401b4ae59b3fb0b676ea (</font><font color="#06989A"><b>HEAD -&gt; </b></font><font color="#4E9A06"><b>1.9</b></font><font color="#C4A000">, </font><font color="#C4A000"><b>tag: v1.9.6rc0</b></font><font color="#C4A000">, </font><font color="#CC0000"><b>origin/1.9</b></font><font color="#C4A000">)</font>
Author: Dane Strandboge &lt;dane@strandboge.com&gt;
Date:   Tue Nov 30 14:46:41 2021 -0600

    chore: upgrade Flux, restore `rules_test.go` (#22948)
    
    * chore: upgrade Flux, restore `rules_test.go` (#22931)
    
    Upgrade Flux to v0.140.0
</pre>